openquake/server/local_settings.py.tools

```diff
@@ -24,4 +27,8 @@ WEBUI_ACCESS_LOG_DIR = '/var/log/oq-engine'
 # system time zone.
 # TIME_ZONE =
 
-DEBUG = False
+# Set debug if the installation is dev mode
+try:
```

Comment:
`DEBUG = os.environ.get('TOOLS_DEV', False) == 'True'`

Reply:
Fixed with this commit https://github.com/gem/oq-engine/commit/ba295d939ba248f2a16d208b58debcc96228e435